server/monitor.ts

```diff
@@ -211,6 +212,16 @@ export const Monitor = new class {
 		return count > 4;
 	}
 
+	/**
+	 * Counts commands that use HTTPs requests. Returns true if too many.
```

Comment:
This documentation isn't accurate. I'm not sure we want to refactor all our monitors to `Chat.ErrorMessage`, though – I think I'd rather this just return true like the others.

Reply:
Returning true it is.

Reply:
Done.